docs/en/deploy/Running-Alluxio-On-a-HA-Cluster.md

```diff
@@ -0,0 +1,359 @@
+commands on each node individually to stop each component. For any node, you can stop a master or worker with:
+
+```bash
+./bin/alluxio-stop.sh master # stops the local master
+./bin/alluxio-stop.sh worker # stops the local worker
+```
+
+### Restart Alluxio
+
+Starting Alluxio is similar. If `conf/workers` and `conf/masters` are both populated, you can start
+the cluster with:
+
+```bash
+./bin/alluxio-start.sh all SudoMount
+```
+
+You can start just the masters and just the workers with the following commands:
+
+```bash
+./bin/alluxio-start.sh masters           # starts all masters in conf/masters
+./bin/alluxio-start.sh workers SudoMount # starts all workers in conf/workers
+```
+
+If you do not want to use `ssh` to login to all the nodes and start all the processes, you can run
+commands on each node individually to start each component. For any node, you can start a master or worker with:
+
+```bash
+./bin/alluxio-start.sh master           # starts the local master
+./bin/alluxio-start.sh worker SudoMount # starts the local worker
+```
+
+### Format the Journal
+
+On any master node, format the Alluxio journal with the following command:
+
+```bash
+./bin/alluxio format
+```
+
+> Formatting the journal will delete all metadata from Alluxio. However, the data in under storage will be untouched.
+
+### Add/Remove Workers Dynamically
+
+Adding a worker to an Alluxio cluster dynamically is as simple as starting a new Alluxio worker
+process, with the appropriate configuration. In most cases, the new worker's configuration should be the same as all
+the other workers' configuration. Run the following command on the new worker to add
+
+```bash
+./bin/alluxio-start.sh worker SudoMount # starts the local worker
+```
+
+Once the worker is started, it will register itself with the Alluxio master, and become part of the Alluxio cluster.
+
+Removing a worker is as simple as stopping the worker process.
+
+```bash
+./bin/alluxio-stop.sh worker # stops the local worker
+```
+
+Once the worker is stopped, and after
+a timeout on the master (configured by master parameter `alluxio.master.worker.timeout`), the master
+will consider the worker as "lost", and no longer consider it as part of the cluster.
+
+### Add/Remove Masters
+
+In order to add a master, the Alluxio cluster must be in HA mode. If you are running the cluster as
+a single master cluster, you must configure it to be an HA cluster to be able to have more than 1
+master.
+
+#### Alluxio HA cluster with embedded journal
+
+When internal leader election is used, Alluxio masters are determined. Adding or removing
+master nodes requires:
+
+* Shut down the whole cluster
+* Add/remove one or more Alluxio master
+* Format the whole cluster
+* Update the cluster-wide embedded journal configuration
+* Start the whole cluster
+
+Note that all previously stored data and metadata in Alluxio filesystem will be erased.
```

Comment:
We should probably note if they are using embedded journal they should not plan to add new masters.

Reply:
done.